guides/keras_cv/classification_with_keras_cv.py

```diff
@@ -0,0 +1,237 @@
+classes = {
+    281: 'tabby, tabby cat',
+    885: 'velvet'
+}
+top_two = [classes[i] for i in top_classes[-2:]]
+print("Top two classes are:", top_two)
+
+"""Great!  Both of these are correct!  But what if you don't care about the
+velvet blanket, and instead only want to know if a cat is in the image or not?
+This can be solved using fine tuning.
+
+# Fine tuning a pretrained classifier
+
+![](https://storage.googleapis.com/keras-nlp/getting_started_guide/prof_keras_intermediate.png)
+
+When labeled images specific to our task are available, fine-tuning a custom
+classifier can improve performance. If we want to train a Cats VS Dogs
+Classifier, using explicitly labeled Cat vs Dog data should perform better than
+the generic classifier data! And for many tasks, no relevant pretrained model
+will be available (e.g., categorizing images specific to your application).
+
+The biggest difficulty when finetuning a KerasCV model is loading and augmenting
+your data.  Luckily, we've handled the second half for you, so all you'll have
+to do is load your own data.
+
+First, let's setup our data pipeline:
+"""
+
+BATCH_SIZE = 32
+AUTOTUNE = tf.data.AUTOTUNE
+tfds.disable_progress_bar()
+
+data, dataset_info = tfds.load("cats_vs_dogs", with_info=True, as_supervised=True)
+train_steps_per_epoch = dataset_info.splits["train"].num_examples // BATCH_SIZE
+train_dataset = data['train']
+
+IMAGE_SIZE = (224, 224)
+num_classes = dataset_info.features["label"].num_classes
+
+random_crop = keras_cv.layers.Resizing(224, 224, crop_to_aspect_ratio=True)
+
+def package_dict(image, label):
+    image = tf.cast(image, tf.float32)
+    image = random_crop(image)
+    label = tf.one_hot(label, num_classes)
+    return {"images": image, "labels": label}
+
+
+train_dataset = train_dataset.shuffle(10 * BATCH_SIZE).map(package_dict, num_parallel_calls=AUTOTUNE)
+train_dataset = train_dataset.batch(BATCH_SIZE)
+
+images = next(iter(train_dataset.take(1)))['images']
+keras_cv.visualization.plot_image_gallery(images, value_range=(0, 255))
+
+"""
+Next, lets assemble a `keras_cv` augmentation pipeline.
+In this guide, we use the standard pipeline
+[CutMix, MixUp, and RandAugment](https://keras.io/guides/keras_cv/cut_mix_mix_up_and_rand_augment/)
+augmentation pipeline.  More information on the behavior of these augmentations
+may be found in their
+[corresponding Keras.io guide](https://keras.io/guides/keras_cv/cut_mix_mix_up_and_rand_augment/).
+"""
+
+augmenter = keras.Sequential(
+    layers=[
+        keras_cv.layers.RandomFlip(),
+        keras_cv.layers.RandAugment(value_range=(0, 255)),
+        keras_cv.layers.CutMix(),
+        keras_cv.layers.MixUp()
+    ]
+)
+
+train_dataset = train_dataset.map(augmenter, num_parallel_calls=tf.data.AUTOTUNE)
+
+images = next(iter(train_dataset.take(1)))['images']
+keras_cv.visualization.plot_image_gallery(images, value_range=(0, 255))
+
+"""
+Next let's construct our model:
+"""
+
+backbone = keras_cv.models.DenseNet121(
```

Comment:
@jbischof and same here?  If I just want the backbone would it be `DenseNet.from_backbone("densenet121_imagenet")`?